CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to `cargo-screeps` and `screeps-game-api`:
+
+1. Ask questions. We're all in on the `#rust-wasm` channel on the [screeps slack].
```

Comment:
> We're all in on the `#rust-wasm`

All-in? Or did you mean _We're all on the `#rust-wasm`_

Reply:
Hm, yeah, the latter. It made sense at the time as in "we're all in [the place that is] on the `#rust-wasm` channel" but that's way less clear than saying "We're all in the `#rust-wasm` channel" or "We're all on the `#rust-wasm` channel".